report/03_methodology/01_project_management.md

```diff
@@ -0,0 +1,26 @@
+## Project management
+
+Overall, the project took place within five weeks and five participants who worked full time on the project. 
+The decisions in the area of project management are described in more detail below.
+
```

Comment:
perhaps add here something like that:

Each team member was assigned a role for which he was responsible. The roles were e.g. 'Git responsible', 'Moderator' or 'Scrum master'.